cloud/README.md

```diff
@@ -0,0 +1,65 @@
+# Please do not publish a solution in either a fork or clone. Keep all of your work local!
+#### If you do want to store your work, we ask that you keep it in a private repo (github lets everyone do this now).
+
+## Introduction
+
+This project is designed to get you familiar with some basic functionalities of our ArchFX Cloud product.
+
+Arch has built a complete framework to help factories monitoring their production lines. Arch FX Cloud is the core of
+this product, as it is the main user interface but also the link with all our other products.
+
+It is basically a webapp served by a Django server, with all the needed backend tools.
+
+To test the skills you will need to be part of the Cloud team, we will build a small API running with Django.
+
+*It should take less than 1 hour to complete. If you think some parts need more, please tell us.*
+
+## Description
+
+The goal of this exercise is to build an API that will allow a user to register time blocks with a name
+and then query the resulting timeline.
+
+1. Setup a Django server within a Docker container. This server will need a PostgreSQL database to store models, which
+   will also be into a Docker container. We recommend describing these containers using docker-compose.
```

Comment:
I think it is helpful to provide more of a framework/skeleton of the project here. Unless you require them to intimately remember off-hand how to make new Dockerfiles and compose files, having the "basic" one present may shave off time and keep candidates moving forward.

So, for that I would suggest providing a Dockerfile and `docker-compose.yml` and `.env`, with the bare essentials, perhaps with comments like `#TODO` indicating where we expect candidates to complete/include their logical solution. 
- I would say at a bare minimum, providing the Dockerfile will be helpful - at the very least it ensures that you are using the same Django version/image as they are!

Reply:
Example structure might be the way I built up the `radiation` folder. I remember David thinking that I made it too easy for candidates, but plenty of them still had trouble with docker, and the ones that didn't were able to give me interesting solutions that were cool even with all that I provided them :)